src/mse.ts

```diff
@@ -188,43 +179,49 @@ export class MSERep extends EventEmitter implements MSE {
 
 	// Rundown basics task
 	async createRundown(profileName: string, playlistID?: string, description?: string): Promise<VRundown> {
-		let playlistExists = false
-		const date = new Date()
-		description = description ? description : `Sofie Rundown ${date.toISOString()}`
+		await this.assertProfileExists(profileName)
+
+		description = description ? description : `Sofie Rundown ${new Date().toISOString()}`
```

Comment:
Depending on whether we get empty strings, this can be simplified to
```suggestion
		description = description ?? `Sofie Rundown ${new Date().toISOString()}`
```